src/getErrors.ts

```diff
@@ -202,10 +202,10 @@ function getLocalErrorFromLine(line: string, projectUrl: string): LocalError | u
     return undefined;
 }
 
-function getBuildSummary({ err, stderr, stdout }: ExecFileResult) {
+function getBuildSummary({ code, signal, stderr, stdout }: SpawnResult) {
     return {
         stdout,
-        hasBuildFailure: !!(err || (stderr && stderr.length && !stderr.match(/debugger/i))), // --inspect prints the debug port to stderr
+        hasBuildFailure: !!(code || signal || (stderr && stderr.length && !stderr.match(/debugger/i))), // --inspect prints the debug port to stderr
```

Comment:
I guess the `||` was always intentional because a `0` success code could still have a signal or stderr output?

Reply:
I could be persuaded that a `0` exit code overrides stderr, but that was the thinking, yes.  Also, I think the code is structured in such a way that `code` will never be 0, but I could be mistaken.